src/seed/seed-db.js

```diff
@@ -5,7 +5,9 @@ import seedMutations from "./seed-mutations";
 import fetch from "node-fetch";
 import { HttpLink } from "apollo-link-http";
 import { InMemoryCache } from "apollo-cache-inmemory";
+import Bluebird from 'bluebird'
 
+fetch.Promise = Bluebird
```

Comment:
who uses this where?

Reply:
The fetch method will then use the Bluebird Promise-Library.